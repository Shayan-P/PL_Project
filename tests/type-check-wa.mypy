checked

a : bool = 3;