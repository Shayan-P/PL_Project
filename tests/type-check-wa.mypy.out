Type Error: wrong assignment type
inner trace: 

(assign-stmt 'a (primitive-type (bool-type)) (num-expr 3))

